dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -111,8 +112,8 @@
     <Import Project="..\Interactions\SliderInteraction\SliderInteraction.vcxitems" Label="Shared" Condition="$(UseInternalSDK) == 'true'" />
     <Import Project="..\MenuFlyout\MenuFlyout.vcxitems" Label="Shared" />
     <Import Project="..\TeachingTip\TeachingTip.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
-    <Import Project="..\RadioMenuFlyoutItem\RadioMenuFlyoutItem.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
-    <Import Project="..\AnimatedVisualPlayer\AnimatedVisualPlayer.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true' and $(UseInsiderSDK) == 'true'" />
+    <Import Project="..\RadioMenuFlyoutItem\RadioMenuFlyoutItem.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" /> 
+    <Import Project="..\AnimatedVisualPlayer\AnimatedVisualPlayer.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true' and $(UseInsiderSDK) == 'true' and $(AnimatedVisualPlayerExcluded) != 'true'" />
```

Comment:
We should fix the use of UseInsiderSDK. Right now we are always building with the 19H1 SDK and UseInsiderSDK is set to true. We should remove the UseInsiderSDK check from AnimatedVisualPlayer, since it should always build. 
At some point UseInsiderSDK will likely mean a 19H2 sdk, so we should keep it around. 

Sorry for all the "cleanup" suggestions, I just want to avoid the "Conditions" getting out of hand.